docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,76 @@
+#### How it worked in zkSync Lite
 
-Firstly, all transactions need to be supported by the priority queue. Users may have their funds locked on an L2 smart contract, and not on their own L2 account. Therefore before moving their funds to L1, they need to send an `Execute` transaction to the zkSync network to release the funds from that smart contract first.
+In the previous version, we only had two operations that could be sent to zkSync Era from L1:
 
-Secondly, the priority queue needs to stay censorship-resistant. But imagine what will happen if users start sending a lot of transactions that take the entirety of the block gas limit? There needs to be a way to prevent spam attacks on the system.
-That's why submitting transactions to the priority queue is no longer free.
-Users need to pay a certain fee to the operator for processing their transactions. It is really hard to calculate the accurate fee in a permissionless way.
-Thus, the fee for a transaction is equal to `txBaseCost * gasPrice`. The `gasPrice` is the gas price of the users' transaction, while `txBaseCost` is the base cost for the transaction, which depends on its parameters (e.g. `gas_limit` for `Execute` transaction).
+- `Deposit` to bridge funds from Ethereum to zkSync.
+- `FullExit` to bridge funds back to Ethereum.
 
-Thirdly, the operator can not commit to processing every transaction within `X` days. Again, this is needed to prevent spam attacks on the priority queue. We changed this rule to the following one:
+If users wanted to deposit or withdraw funds, they had to send a transaction request to the smart contract. The request was appended to the priority queue. The queue had the following rules:
 
-- The operator must do at least `X` amount of work (see below) on the priority queue or the priority queue should be empty.
+1. All transactions were processed sequentially, i.e. FIFO.
+2. Each priority operation had to be processed by the operator within `X` days from submission to the contract.
 
-In other words, we require the operator to do its best instead of requiring a strict deadline. The measure of "the work" is still to be developed. Most likely it will be the number of `gas` the priority operations used.
+The first rule was strictly enforced by the smart contract. 
 
-In the future, we will also add the ability to "prioritize" L1->L2 transactions, allowing users to speed up the inclusion of their transactions in exchange for paying a higher fee to the operator.
+The second rule could be violated if the operator became malicious or unavailable. If that happened, the system entered **exodus mode**, where no new blocks were processed and users could withdraw their funds without operator interference.
 
-## Priority mode
+#### How it works in zkSync Era
```

Comment:
I don't see it anyway, but we should state either here or on top, that this is not implemented -- it's just a design description

@kailash-manasarovar please make sure to add this before merging

Reply:
I removed all Priority Mode stuff.

Reply:
Are you saying that the whole description in the **How it works in zkSync Era** section has not been implemented?

Reply:
I certainly do

Reply:
@barakshani I think you are confusing Priority Mode (not implemented) with Priority Queue (Yes, implemented) ... I removed all Priority Mode stuff for that reason but kept the PQ.